react-ui/src/pages/home/Home.jsx

```diff
@@ -88,6 +90,7 @@ function Home() {
     <>
       <AppBarDivided color="default" position="sticky" />
       <Container>
+        <FeatureBanner screen={SCREENS.HOME} />
```

Comment:
Instead of passing the screen in, why not use `location.path` so the feature banner can find out for itself what to render? i.e. the path can be the key?

Reply:
Ya that's a good idea, though I'll keep screen in there as an override just in case